src/SDKs/HDInsight/Management.HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Resources;
+using System.Runtime.CompilerServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Azure .NET SDK")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+[assembly: AssemblyTitle("Microsoft Azure HDInsight Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure HDInsight.")]
+[assembly: AssemblyVersion("3.1.0.0")]
+[assembly: AssemblyFileVersion("3.1.0.0")]
+
+[assembly: InternalsVisibleTo("Microsoft.Azure.Management.HDInsight.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
@wawon-msft any reason you need this?
What scenarios do you have for which you need internals visible? Are you writing API tests only?

Reply:
There is a helper method in CreateParametersConverter called GetNodeSize, which to my understanding is the only time we requires access to internals.